source/quickstart.rst

```diff
@@ -113,13 +123,48 @@ Stormpath also can do a lot more (like :ref:`Groups <group-mgmt>`, :ref:`Multite
 
         pip install stormpath
 
-    If you'd like to update to use the latest Stormapth Python SDK, you can
+    If you'd like to update to use the latest Stormpath Python SDK, you can
     instead run:
 
     .. code:: console
 
         pip install --upgrade stormpath
 
+.. only:: java
+
+    Installing the SDK
+    ------------------
+
+    To setup up your environment for this quickstart, follow these steps:
+
+    Include the following  dependencies in your Maven ``pom.xml`` file:
+
+    .. code-block:: xml
+
+        ...
+
+        <dependencies>
+
+            ...
+
+            <dependency>
+                <groupId>com.stormpath.sdk</groupId>
+                <artifactId>stormpath-sdk-api</artifactId>
+                <version>###latest_stormpath_version###</version>
```

Comment:
This gets automatically updated with the actual latest version by the javascript in `javascript_for_java.js`